cherrypy/test/test_static.py

```diff
@@ -215,13 +215,13 @@ def test_config_errors(self):
         self.assertErrorPage(500)
         if sys.version_info >= (3, 3):
             errmsg = (
-                'TypeError: staticdir\(\) missing 2 '
+                r'TypeError: staticdir\(\) missing 2 '
```

Comment:
This change does not change the message string content in any way. If the backslash prefixing was necessary before, it still is, and the other way around.

```
$ python3
Python 3.5.3 (default, May 10 2017, 15:05:55) 
[GCC 6.3.1 20161221 (Red Hat 6.3.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 'TypeError: staticdir\(\) missing 2 '
'TypeError: staticdir\\(\\) missing 2 '
>>> r'TypeError: staticdir\(\) missing 2 '
'TypeError: staticdir\\(\\) missing 2 '
>>> 
```